src/tooltipDisplay.ts

```diff
@@ -31,9 +31,18 @@ export function bindData(tooltipPlaceholder: Selection<Element | EnterElement |
   tooltipRows.exit().remove();
 
   const row = tooltipRows.enter().append('tr')
-    .attr('class', 'tooltip-row');
-  row.append('td').attr('class', 'key').text(function (d: TooltipData) { return d.title + ':'; });
-  row.append('td').attr('class', 'value').text(function (d: TooltipData) { return d.value; });
+    .attr('class', 'tooltip-row')
+    .each(function(d) {
+      const sel = select(this);
+      if (d.render) {
```

Comment:
Sorry if this comes a bit late -- I wonder if the customization should happen at the `tr` level above?  

(I'm thinking about a case where users want to customize the global wrapper to be `div` instead of `table` -- then the rendered row doesn't have to be `tr` with `td` inside anymore.)

Reply:
I placed the `render` injection within `<td>`s because placing `<div>`s elsewhere in a `<table>` is not valid. ([MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table) -- See "permitted content"; [CSS-Tricks](https://css-tricks.com/using-divs-inside-tables/))

I tried to get this customization to support swapping out the entire `<table>`, in order to use non-tabular rendering. But I wasn't able to find a good way given current constraints to maintain the data binding -- this functionality would require more thorough changes to how the lib renders and binds data and I was trying to restrain myself.

Might be worth another pass at this before cutting a new release, or marking the `render` API as experimental and subject to change, to increase the flexibility it offers. We could possibly modify this so that if the `render` prop exists, the wrapper is rendered as a `<div>` rather than a `<table>`, and then inject `render`ed content into a series of `div.tooltip-row` elements.